### ISSUE TITLE
Request for Auto complete feature

### ISSUE THREAD

**Issue by manubalasree** [sarthology/caligator#33]
An auto complete / suggestion feature would make its easy to key in the units. Currently its a bit confusion to figure out what notation works. Example for miles mi works but its not easy to find out .

**Comment by DRFR0ST** [comment]
Agree, it's pretty hard and confusing at the beginning. Built-in examples and/or auto-complete would be really helpful.
